packages/flutter/lib/src/gestures/velocity_tracker.dart

```diff
@@ -195,7 +215,7 @@ class VelocityTracker {
       final double age = (newestSample.time - sample.time).inMicroseconds.toDouble() / 1000;
       final double delta = (sample.time - previousSample.time).inMicroseconds.abs().toDouble() / 1000;
       previousSample = sample;
-      if (age > _horizonMilliseconds || delta > _assumePointerMoveStoppedMilliseconds) {
+      if (age > _horizonMilliseconds || delta > VelocityTracker._assumePointerMoveStoppedMilliseconds) {
```

Comment:
same as the styling question above, either both `_horizonMilliseconds` and `_assumePointerMoveStoppedMilliseconds` prefixed with class name or none